src/render.tsx

```diff
@@ -135,18 +136,31 @@ function updateWithAct(
 }
 
 interface DebugFunction {
-  (message?: string): void;
+  (options?: DebugOptions | string): void;
   shallow: (message?: string) => void;
 }
 
 function debug(
   instance: ReactTestInstance,
   renderer: ReactTestRenderer
 ): DebugFunction {
-  function debugImpl(message?: string) {
+  function debugImpl(options?: DebugOptions | string) {
+    const { defaultDebugOptions } = getConfig();
     const json = renderer.toJSON();
```

Comment:
style, nit:
Pls move `const json = ...` line just above the `if (json)` statement, as it is not used during option calculation part.